docker-compose.yml

```diff
@@ -1,7 +1,12 @@
 version: '2'
 services:
   autopush:
-    image: bbangert/autopush
+    build: .
+    # If you do not wish to use a local build, comment out
+    # the previous 'build:' line and uncomment the following
+    # 'image:' line. This will use the latest production
+    # built image.
+    #image: mozilla/autopush
```

Comment:
I'll note that the current mozilla/autopush image doesn't work with this version of docker-compose, thus the comment for now. Obviously, the most stable thing is to build locally, but if we want folks to just play, we can switch the defaults here.